### DISCUSSION TITLE
Questions about result files of RTA mode

### DISCUSSION THREAD

**Discussion by Iridium-191** [ttadano/alamode#134] [Q&A] [answered]
Dear Tadano:

While calculating thermo-conductivity using RTA mode of "anphon" module, we get a '.result' file. I wonder what is the unit and physical meanings of coordinates at the section head of "#GAMMA_EACH" section? Is it group velocity? 
For example:
`#GAMMA_EACH
28 16
12
      **0.0100995        974.529       -1574.75
     -0.0100995       -974.529        1574.75
       -843.971       -487.251       -1574.76
        843.971        487.251        1574.76
    -0.00907661       -974.531       -1574.75
     0.00907662        974.531        1574.75
        843.966        487.256       -1574.75
       -843.966       -487.256        1574.75
       -843.981        487.244       -1574.75
        843.981       -487.244        1574.75
        843.979       -487.244       -1574.75
       -843.979        487.244        1574.75**
    0.000173789
    0.000177937` 
    Another question is the unit of phonon relaxation time. In the website of alamode, the paragraph describing the '.result' file says that is 'phonon linewidth'. I also noticed that this number increased significantly with increasing temperature, so it maybe in the unit of energy, frequency or wavenumber? I'm very concerned about the units used here.
Looking forward to your answer and best wishes!

Faeser

**Comment by ttadano** [comment, marked as answer]
Let me explain using the following example:

```
#GAMMA_EACH
105 10
12
       -197.596       -342.247       -451.174
        197.596        342.247        451.174
        395.192   -8.27506e-05       -451.174
       -395.192    8.27429e-05        451.174
        197.596        342.247       -451.174
       -197.596       -342.247        451.174
       -197.596        342.247       -451.174
        197.596       -342.247        451.174
        197.596       -342.247       -451.174
       -197.596        342.247        451.174
       -395.192     8.2739e-05       -451.174
        395.192   -8.27467e-05        451.174
       0.368313
       0.368314
       0.368343
       0.368471
```

The numbers `105 10` are the indices for the irreducible **q** point and the phonon branch, respectively. 

The number `12` is the number of **q**<sup>★</sup> (star of **q**), which can be generated from **q** by applying the space group operations. 

The following 12x3 data represent the group velocities of these `12` **q**<sup>★</sup> points in the Cartesian coordinate, (v<sup>x</sup>, v<sup>y</sup>, v<sup>z</sup>). The unit is **m/s**.

After that, the phonon linewidths (half-width, imaginary part of the self-energy, Γ<sub>q</sub>(T)) computed at the given temperature grid are shown. The unit is **cm<sup>-1</sup>**.
The linewidth data (cm<sup>-1</sup>) can be converted into the lifetime τ<sub>q</sub> (ps) as τ<sub>q</sub> = 1/(2Γ<sub>q</sub>(T)) * 10<sup>12</sup> / (2π\*c\*100), where c=299792458 m/s is the speed of light.

Note:
If the phonon mode is degenerate, the Γ<sub>q</sub>(T) values need to be averaged over the degenerate branches if `TRISYM=1` (default).

**Reply by Iridium-191** [reply]
Great, I get it！Thank you for your help and patience！
